### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by bohdan-mykhailenko**
[DEMO LINK](https://bohdan-mykhailenko.github.io/react_dynamic-list-of-todos/)

### CODE REVIEW

**Review by serentinos — APPROVED** [review]
Good job!

File: src/App.tsx [hunk 1]
```diff
@@ -17,18 +64,35 @@ export const App: React.FC = () => {
...
+              {isLoading
+                ? (
+                  <Loader />
+                ) : (
+                  <TodoList
+                    todos={visibleTodos}
+                    selectedTodoId={selectedTodoId}
+                    onSelectTodoId={selectTodoId}
+                  />
+                )}
```
Comment (33Orange) [inline src/App.tsx:84]:
```suggestion
              {isLoading
                ? <Loader />
                : (<TodoList
                    todos={visibleTodos}
                    selectedTodoId={selectedTodoId}
                    onSelectTodoId={selectTodoId}
                  />)
               }
```

File: src/components/TodoModal/TodoModal.tsx [hunk 2]
```diff
@@ -1,49 +1,88 @@
...
+                {title}
+              </p>
+
+              <p className="block" data-cy="modal-user">
+                {completed
+                  ? (
+                    <strong className="has-text-success">Done</strong>
+                  ) : (
+                    <strong className="has-text-danger">Planned</strong>
+                  )}
```
Comment (33Orange) [inline src/components/TodoModal/TodoModal.tsx:75]:
```suggestion
                {completed
                  ? <strong className="has-text-success">Done</strong>
                  : <strong className="has-text-danger">Planned</strong>
                }
```

**Review by 33Orange — APPROVED** [review]
Good job

**Pull request closed without merging**